backend/src/main/java/com/crm/model/db/AccountEntity.java

```diff
@@ -14,9 +13,13 @@
 import java.sql.Timestamp;
 
 @Entity
-@Table(name = "accounts")
-@Getter
 @NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Getter
+@Setter
+@EqualsAndHashCode
+@Builder
```

Comment:
I tu

Reply:
To nie zadziała z Builderem, zostawiłem jak było